octobot_trading/exchanges/exchange_manager.py

```diff
@@ -151,13 +153,16 @@ def reset_exchange_personal_data(self):
     async def _create_simulated_exchange(self):
         self.exchange = ExchangeSimulator(self.config, self.exchange_type, self, self.backtesting_files)
         await self.exchange.initialize()
+        await self._initialize_simulator_time_frames()
+        self.exchange_config.set_config_time_frame()
         self.exchange_config.set_config_traded_pairs()
         await self._create_exchange_channels()
 
     async def _init_simulated_exchange(self):
         try:
             await self.exchange.modify_channels()
             await self.exchange.create_backtesting_exchange_producers()
+            await start_backtesting(self.exchange.backtesting)
```

Comment:
Why this is not made in `ExchangeSimulator` ? Maybe an `ExchangeSimulator.start()` or `.start_backtesting()` ?